executor/adapter.go

```diff
@@ -81,13 +83,18 @@ func (a *statement) IsDDL() bool {
 	return a.isDDL
 }
 
+// Exec implements the ast.Statement Exec interface.
+// This function builds an Executor from a plan, if the Executor doesn't return result,
```

Comment:
->This function builds an Executor from a plan. If the Executor doesn't return result,